cmake/FindIgnOGRE2.cmake

```diff
@@ -53,6 +53,14 @@ if (${IgnOGRE2_FIND_VERSION_MAJOR})
   endif()
 endif()
 
+if (${IgnOGRE2_FIND_VERSION_MINOR} VERSION_EQUAL "2")
```

Comment:
Thinking on the future and to avoid the assumption that not 2 is 1, could work something like: 
```cmake
message(STATUS "-- Finding OGRE 2.${IgnOGRE2_FIND_VERSION_MINOR}")
set(OGRE2_INSTALL_PATH "OGRE-2.${IgnOGRE2_FIND_VERSION_MINOR}")
```
instead of the whole if block?

Reply:
Makes sense